KitechRW/kitech
# Quick fix

#### what does this PR do?
- apply changes and feedback
#### Description of task to be completed
- fix typo "Amzina to Amazina"
- Fix order images that was stratching on mobile view
- Update pricing and mention that the price is per year
#### How should this be manually tested
- visit order form
- visit home page and check pricing
#### Any background context you want to provide

#### What are the relevant trello story link

#### Screenshots (if appropriate)

#### Questions:
